lib/fragment.js

```diff
@@ -16,6 +16,20 @@ const hasValue = value => {
     return false;
 };
 
+const getFragmentAssetUris = (refs, assetSize) => {
+    const scriptUris = [];
+    const styleUris = [];
+
+    for (const ref of refs) {
+        if (ref.rel === 'fragment-script') {
```

Comment:
Do we have tests for this filtering behaviour?

Reply:
Yes, tests/tailor.js.